Validation/Configuration/python/autoValidation.py

```diff
@@ -6,3 +6,4 @@
                    'standardValidation' : ['prevalidation','validation','validationHarvesting']
                  }
 
+autoValidation['phase2Validation'] = autoValidation['baseValidation']+autoValidation['trackingOnlyValidation']+autoValidation['muonOnlyValidation']
```

Comment:
apparently this syntax doesn't work. cmsDriver output with this auto ends up having only the first element baseValidation, while everything else is lost.